src/main/java/com/mraof/minestuck/blockentity/redstone/StructureCoreBlockEntity.java

```diff
@@ -33,7 +32,7 @@ public class StructureCoreBlockEntity extends BlockEntity
 	private ActionType actionType;
 	private int shutdownRange = 32;
 	private boolean hasWiped = false;
-	public final Structure[] SOLVABLE_STRUCTURES = new Structure[]{MSConfiguredStructures.FROG_TEMPLE.get()};
+	public final Structure[] SOLVABLE_STRUCTURES = new Structure[]{};
```

Comment:
Why remove frog temples?

Reply:
I got the impression that the feature was already not working as it should (since it was getting the structure from the wrong registry), as well as being an incomplete and unused feature. When I then changed `MSConfiguredStructures.FROG_TEMPLE` from a registry object to a resource key, I didn't bother changing/fixing it for the previously mentioned reasons, and instead just removed the structure from the array.